ckanext/msl_ckan/schemas/datasets/cd2_data.collection.yml

```diff
@@ -25,22 +30,26 @@ dataset_fields:
   label: wave
   repeating_subfields:
   - field_name: wave_title
-    label: Short identifying title
-  - field_name: wave_measurement_stages # e.g. 1 month, 3 months, etc; can we somehow harmonize these accross cohorts?
-    label: Stage of measurement
+    label: Short identifying title of the wave
+  # DH: measurement stage is not in the metadata model, corresponds with Universe
+  #- field_name: wave_measurement_stages # e.g. 1 month, 3 months, etc; can we somehow harmonize these accross cohorts? 
+  #  label: Stage of measurement (e.g., target age of participants in the wave)
   - field_name: notes # standard CKAN field; abstract description of the wave
-    label: Short description of the wave
+    label: Short description of the wave (Abstract)
   - field_name: wave_universe # textual description of universe; redundant with description field available?
-    label: Wave universe description
+    label: Short summary of the participants in the cohort (e.g., target age) 
   - field_name: wave_publisher_name
-    label: Publisher of the wave data
+    label: Name of organization hosting the wave data (e.g. university, library, repository, etc.)
     required: true
   - field_name: wave_publisher_identifier
-    label: Publiser ID
+    label: Identifier of the organization hosting the wave data
   - field_name: wave_publisher_identifier_type
-    label: Publisher ID-type
-  - field_name: wave_lifestage
-    label: Life stage
+    label: Type of organization identifier for publisher of the wave data
+  # DH: This info is contained within Universe (was included in older model versions)
+  #- field_name: wave_lifestage
+  #  label: Life stage
+  # DH: SpatialCoverage consists of CountryCode, GeographicLocationReference and Lowest/Higest
+  #     reference in the metadata model. I guess these will be collapsed into 1?
```

Comment:
If we want to make use of [CKAN geospatial features](https://ckan.org/features/geospatial/) a textual geographical description is not enough. A bounding box specifying the geographical area surveyed should be included.

Reply:
We don't have coordinates and a unambiguous mapping does not seem feasible.